### ISSUE TITLE
Dynamical mean-field theory for stochastic gradient descent in Gaussian mixture classification

### ISSUE THREAD

**Issue by Hiroki11x** [Hiroki11x/Papers#24]
<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Analyzing stochastic-gradient descent in neural networks at small sample complexity, down to a constant, for a highly non-convex loss? Impossible? No more! Particularly happy about this one :) arxiv:2006.06098 <a href="https://t.co/VLTxNaspi0">pic.twitter.com/VLTxNaspi0</a></p>&mdash; Lenka Zdeborova (@zdeborova) <a href="https://twitter.com/zdeborova/status/1271337201884958720?ref_src=twsrc%5Etfw">June 12, 2020</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>

We analyze in a closed form the learning dynamics of stochastic gradient descent (SGD) for a single layer neural network classifying a high-dimensional Gaussian mixture where each cluster is assigned one of two labels. This problem provides a prototype of a non-convex loss landscape with interpolating regimes and a large generalization gap. We define a particular stochastic process for which SGD can be extended to a continuous-time limit that we call stochastic gradient flow. In the full-batch limit we recover the standard gradient flow. We apply dynamical mean-field theory from statistical physics to track the dynamics of the algorithm in the high-dimensional limit via a self-consistent stochastic process. We explore the performance of the algorithm as a function of control parameters shedding light on how it navigates the loss landscape.

[DeepL]
各クラスタに2つのラベルのいずれかが割り当てられている高次元ガウス混合物を分類する単層ニューラルネットワークのための確率的勾配降下(SGD)の学習ダイナミクスをクローズドな形式で解析する。この問題は、補間領域と大きな一般化ギャップを持つ非凸の損失ランドスケープのプロトタイプを提供する。我々は、SGDを連続時間限界まで拡張できる特定の確率過程を定義し、これを確率勾配流と呼ぶ。フルバッチ限界では、標準的な勾配流を回復する。統計物理学の動的平均場理論を適用して、自己矛盾しない確率過程を介して高次元限界におけるアルゴリズムのダイナミクスを追跡します。制御パラメータの関数としてのアルゴリズムの性能を探り、アルゴリズムがどのように損失ランドスケープをナビゲートするかを明らかにする。

![Screen Shot 2020-06-13 at 10 38 24](https://user-images.githubusercontent.com/8721858/84556987-13465c80-ad62-11ea-8be7-f43212242f0b.png)
